lineman/app/components/navbar/navbar_user_options.coffee

```diff
@@ -3,5 +3,7 @@ angular.module('loomioApp').directive 'navbarUserOptions', ->
   restrict: 'E'
   templateUrl: 'generated/components/navbar/navbar_user_options.html'
   replace: true
-  controller: 'NavbarController'
+  controller: ($scope, DiscussionFormService) ->
```

Comment:
This was broken for me (it still is..). I don't know why. I think it's a bower package thingy, but haven't investigated it yet.